frontend/src/styles/_global.scss

```diff
@@ -27,3 +27,7 @@ body {
   border: 1px solid $secondary-1;
   background-color: $secondary-1;
 }
+
+.text-bold {
+  font-weight: bold;
+}
```

Comment:
Nit: it's not necessary to introduce a new class. [Boostrap built-in classes](https://getbootstrap.com/docs/4.0/utilities/text/#font-weight-and-italics) can be used.

Reply:
Done!